app/scripts/modules/kubernetes/serverGroup/configure/wizard/volumes.html

```diff
@@ -116,6 +116,30 @@
         <input type="text" class="form-control input-sm" name="path" ng-model="source.secret.secretName"/>
       </div>
     </div>
+    <div class="form-group" ng-show="source.type === 'AWSELASTICBLOCKSTORE'">
```

Comment:
Prefer `ng-if` to `ng-show` in general, but if the rest of this file uses `ng-show`, it's fine to keep it.